examples/util/tflite/TFliteModelImporter.js

```diff
@@ -192,9 +193,14 @@ class TFliteModelImporter {
       this._compilation.setPreference(getPreferCode(this._backend, this._prefer));
       await this._compilation.finish();
       this._execution = await this._compilation.createExecution();
-
+      
       const outputSize = graph.tensors(outputs[0]).shapeArray().reduce((a,b)=>a*b);
-      const outputTensor = new Float32Array(outputSize);  
+      let outputTensor;
+      if (this.isQuantized) {
```

Comment:
**this.isQuantized** is not defined, please replace it by **this._isQuantized**, and this **this._isQuantized** variable should also be defined in  OnnxModelImporter.js and OpenVINOModelImporter.js.